libr/socket/run.c

```diff
@@ -865,11 +876,7 @@ R_API int r_run_config_env(RRunProfile *p) {
 				is_child = true;
 
 				if (p->_dofork && !p->_dodebug) {
-#ifdef _MSC_VER
-					int child_pid = r_sys_fork ();
-#else
 					pid_t child_pid = r_sys_fork ();
```

Comment:
`pid_t` is not necessary and this should be `int child_pid`, because `r_sys_fork` returns int.